content/tutorials/r_gbif_name_matching/index.Rmd

```diff
@@ -12,70 +12,90 @@ output:
-    gbif_species_name_match(name_col = "name") 
+    gbif_species_name_match(name = "name") 
+```
+
+The `name` argument is the column name containing the scientific names. Default value: `name`. So, the code above is equivalent to:
+
+```{r similar_code, eval=FALSE}
+species_list_matched <- species_list %>% 
+    gbif_species_name_match() 
 ```
 
-The `name_col` argument is the column name containing the scientific names. 
+By default, `gbif_species_name_match` returns the following GBIF fields: `usageKey`, `scientificName`, `rank`, `order`, `matchType`, `phylum`, `kingdom`, `genus`, `class`, `confidence`, `synonym`, `status`, `family`.
 
 Take a look at the updated data:
 
-```{r}
-knitr::kable(head(species_list_matched))
+```{r view_output}
+knitr::kable(species_list_matched)
 ```
 
-You can also specify which info (`gbif_terms`) GBIF should return. For example, return a lot of terms: 
+Notice that GBIF fields whose name is already used as column name are automatically renamed by adding suffix  `1`. In our case, input data.frame `species_list` contains already a column called `kingdom`: the GBIF kingdom values are returned in column `kingdom1`:
+
+```{r zoom_kingdom}
+species_list_matched %>% select(kingdom, kingdom1)
+```
+
+You can specify which GBIF fields you would like to have:
 
 ```{r, message=FALSE, warning=FALSE}
 species_list %>% 
-    gbif_species_name_match(name_col = "name",
-                            gbif_terms = c('usageKey', 'scientificName', 
-                                           'rank', 'order', 'matchType', 
-                                           'phylum', 'kingdom', 'genus', 
-                                           'class', 'confidence', 'synonym', 
-                                           'status', 'family')) %>%
-    head() %>%
+    gbif_species_name_match(
+      gbif_terms = c(
+        'scientificName', 
+        'family',
+        'order',
+        'rank',
+        'matchType',
+        'confidence',
+        'status')) %>%
     kable()
 ```
 
-or rather just a subset:
+As `gbif_species_name_match` is a wrapper around rgbif function `name_backbone`, the authors thought important to offer the possibility to pass any parameter of rgbif function `name_backbone`. So, for example, you can set `strict = TRUE` to fuzzy match only the given names, but never a taxon in the upper classification:
 
-```{r, message=FALSE, warning=FALSE}
+```{r example_strict, message=FALSE, warning=FALSE}
 species_list %>% 
-    gbif_species_name_match(name_col = "name",
-                            gbif_terms = c('scientificName', 'family',
-                                           'order', 'matchType')) %>%
-    head() %>%
+    gbif_species_name_match(strict = TRUE) %>%
     kable()
 ```
 
+All accepted parameters of `name_backbone`: 'rank', 'kingdom', 'phylum', 'class', 'order', 'family', 'genus', 'strict', 'verbose', 'start', 'limit', 'curlopts'. See `?name_backbone` for more details.
```

Comment:
a function has arguments, not parameters.

Reply:
Thanks!